tests/rhaiis/test_ci.py

```
@@ -0,0 +1,82 @@
+
+        assert result.exit_code == 0
+        assert "DRY-RUN" in result.output
+        assert "RHOAI" in result.output
+
+
+class TestCiTest:
+    """Tests for ci test command."""
+
+    @pytest.fixture
+    def runner(self):
+        return CliRunner(env={"FORGE_ARTIFACT_DIR": "/tmp/artifacts"})
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/artifacts"

(S108)